apps/ai/src/app/api/receive/route.ts

```diff
@@ -90,12 +91,13 @@ export const POST = async (request: NextRequest) => {
   context.userId = user.id.toString();
 
   // Pre-fetch data relevant to most bookings.
-  const [eventTypes, availability] = await Promise.all([
+  const [eventTypes, availability, users] = await Promise.all([
     fetchEventTypes(),
     fetchAvailability({
       dateFrom: now,
       dateTo: now,
     }),
+    extractUsers(`${parsed.text} ${parsed.subject}`),
```

Comment:
Should subject be before text? Also, will users have to mention their/others' Cal usernames in emails for this to work? What about using original `to` email instead?